src/partials/products.html

```diff
@@ -0,0 +1,30 @@
+<section class="section__products">
```

Comment:
<section class="section__porducts"> 
zmień na
<section class="porducts"> bo to będzie nazwa bloku a reszta poniżej będzie jego elementami np.
products --> products__subtitle itd.